src/status.rs

```diff
@@ -1780,18 +829,41 @@ impl StatusClass {
     /// assert_eq!(status, BadRequest);
     /// // So now let’s treat it as that.
     /// ```
-    #[inline]
+    /// All status codes that do not map to an existing status class are matched
+    /// by a `NoClass`, variant that resolves to 200 (Ok) as default code.
+    /// This is a common handling for unknown status codes in major browsers.
```

Comment:
I haven't heard of this, but I've not really looked at the status code part of implementations